HW_04/Task01.py

```diff
@@ -0,0 +1,25 @@
+# Создайте несколько переменных заканчивающихся и не оканчивающихся на «s». Напишите функцию,
+#  которая при запуске заменяет содержимое переменных оканчивающихся на s (кроме переменной из одной буквы s) на None. 
+# Значения не удаляются, а помещаются в одноимённые переменные без s на конце.
+
+def func() -> None:
+    glob = globals()
+    dict = {}
+    for key, value in glob.items():
```

Comment:
Либо можно взять кортеж, и тогда не придется определять словарь `dict`
```suggestion
    for key, value in tuple(glob.items()):
```